push-images.sh

```diff
@@ -14,13 +14,19 @@ docker push cimg/elixir:1.13-erlang-23.3-browsers
 docker push cimg/elixir:1.13.4-erlang-23.3-browsers
 docker push cimg/elixir:1.13-erlang-24.3
 docker push cimg/elixir:1.13.4-erlang-24.3
-docker tag cimg/elixir:1.13.4-erlang-24.3 cimg/elixir:1.13.4
-docker tag cimg/elixir:1.13-erlang-24.3 cimg/elixir:1.13
 docker push cimg/elixir:1.13-erlang-24.3-node
 docker push cimg/elixir:1.13.4-erlang-24.3-node
-docker tag cimg/elixir:1.13.4-erlang-24.3-node cimg/elixir:1.13.4-node
-docker tag cimg/elixir:1.13-erlang-24.3-node cimg/elixir:1.13-node
 docker push cimg/elixir:1.13-erlang-24.3-browsers
 docker push cimg/elixir:1.13.4-erlang-24.3-browsers
-docker tag cimg/elixir:1.13.4-erlang-24.3-browsers cimg/elixir:1.13.4-browsers
-docker tag cimg/elixir:1.13-erlang-24.3-browsers cimg/elixir:1.13-browsers
+docker push cimg/elixir:1.13-erlang-25.0
+docker push cimg/elixir:1.13.4-erlang-25.0
+docker tag cimg/elixir:1.13.4-erlang-25.0 cimg/elixir:1.13.4
+docker tag cimg/elixir:1.13-erlang-25.0 cimg/elixir:1.13
+docker push cimg/elixir:1.13-erlang-25.0-node
+docker push cimg/elixir:1.13.4-erlang-25.0-node
+docker tag cimg/elixir:1.13.4-erlang-25.0-node cimg/elixir:1.13.4-node
+docker tag cimg/elixir:1.13-erlang-25.0-node cimg/elixir:1.13-node
+docker push cimg/elixir:1.13-erlang-25.0-browsers
+docker push cimg/elixir:1.13.4-erlang-25.0-browsers
+docker tag cimg/elixir:1.13.4-erlang-25.0-browsers cimg/elixir:1.13.4-browsers
+docker tag cimg/elixir:1.13-erlang-25.0-browsers cimg/elixir:1.13-browsers
```

Comment:
Does this have the latest shared-tools? It's missing the lines where we assume the parent tag. For example, something like this:

docker push cimg/elixir:1.13

Reply:
oh gosh. i think i only did it for the alias and not the default parent tag

Reply:
this is the PR for it. i'll respin after this has been merged and updated on this repo: https://github.com/CircleCI-Public/cimg-shared/pull/67